consul/consul_test.go

```diff
@@ -83,7 +88,7 @@ func TestNewConsulRegister(t *testing.T) {
 	assert.NotNil(t, consulRegister)
 }
 
-// TestNewConsulResolver tests the NewConsulResolver function with check option.
+// TestNewConsulResolver tests the NewConsulRegister function with check option.
```

Comment:
What is the purpose of changing this? Seems that the keyword has not been completely changed?